### PR TITLE
 Routers Completed

### PR DESCRIPTION
**Pull request by 002sathwik**
added and tested  all routers excluding, users, added mailing certificate , and activity points , updation with mailing user , if his gets new activity points

### CODE REVIEW

File: prisma/schema.prisma
Comment (Prabhuomkar9) [inline prisma/schema.prisma:1]: use StartCase

File: public/flc_DB_Schema.png
Comment (Prabhuomkar9) [inline public/flc_DB_Schema.png:1]: remove

File: src/components/Test.tsx
Comment (Prabhuomkar9) [inline src/components/Test.tsx:1]: temporary test file

File: src/pages/index.tsx
Comment (Prabhuomkar9) [inline src/pages/index.tsx:1]: temporary test code

File: src/server/auth.ts
Comment (Prabhuomkar9) [inline src/server/auth.ts:1]: typeof role = UserRole from prisma

File: src/utils/certificates/Topperformer_3765a3f2-0785-4d4d-bcf1-6fc7dc9144e7.png
Comment (Prabhuomkar9) [inline src/utils/certificates/Topperformer_3765a3f2-0785-4d4d-bcf1-6fc7dc9144e7.png:1]: remove

File: src/server/schema/zod-schema.ts
Comment (Prabhuomkar9) [inline src/server/schema/zod-schema.ts:1]: seperate files is better

**Review by Prabhuomkar9 — CHANGES_REQUESTED** [review]
Check comments and do the changes

File: src/utils/helper/findEventById.ts
Comment (satwikrprabhu) [inline src/utils/helper/findEventById.ts:1]: create one helper.ts file and export all the functions from there

**Pull request merged**